src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -4,9 +4,20 @@
   align-items: center;
   height: 100%;
   width: 100%;
-}
 
-.remote-media video {
-  width: 100%;
-  height: 100%;
+  span {
+    margin: auto;
+    padding: 0.4rem;
+
+    div {
+      &:last-child {
```

Comment:
is there a reason to apply it to the last child instead of the `div` itself?

Reply:
I will use `div` as it's more consistent this way.